index.html

```diff
@@ -2,22 +2,76 @@
 <html lang="en">
 
 <head>
-
    <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&family=Roboto&display=swap" rel="stylesheet">
-   <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&family=Roboto:wght@400;500&display=swap"
+   <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&family=Roboto:wght@400;500&display=swap">
+   <link href="https://fonts.googleapis.com/css2?family=Amatic+SC&family=Roboto:wght@300;400&display=swap"
       rel="stylesheet">
    <link rel="stylesheet" href="css/styles.css">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Title</title>
+
 </head>
 
+
 <body class="body">
-   <div class="container">
-      <section class="section-sightseens">
-         <h1 class="main-title">Достопремечательности</h1>
+
+   <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+   <script type="text/javascript" src="/js/script.js"></script>
+   <header class="header">
+      <div class="continer">
+         <div class="header__body">
+            <a href="" class="header__logo">
+               <img src="/img/logo.png" alt="">
+            </a>
+            <div class="header__burger">
```

Comment:
Если это кнопка то должно быть так

```suggestion
            <button class="header__burger" type="button">
```